openedx/features/ondemand_email_preferences/tests/test_views.py

```diff
@@ -0,0 +1,50 @@
+"""
+All tests for ondemand preferences views.
+"""
+import mock
+from django.test import Client
+from django.urls import reverse
+from rest_framework import status
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
```

Comment:
Please run isort

Reply:
I already did, this is after applying isort